components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/test/resources/repository/conf/identity/identity-outbound-adapter.properties

```diff
@@ -25,3 +25,8 @@ adapter.httppublisher.defaultMaxConnections=20.0
 adapter.httppublisher.defaultMaxConnectionsPerRoute=20.0
 adapter.httppublisher.sslCertificateAlias=
 adapter.httppublisher.type=Publisher
+adapter.httppublisher.maxRetries=2
+adapter.httppublisher.ioThreadCount=5
+adapter.httppublisher.executorCorePoolSize=5
```

Comment:
Lets merge this once considering the renaming suggestion here - 
https://github.com/wso2-extensions/identity-event-publishers/pull/43#discussion_r2299791326